src/Nonlinear/evaluator.jl

```diff
@@ -13,20 +13,46 @@ end
 
 Return the 1-indexed value of the constraint index `c` in `evaluator`.
 
-## Examples
-
-```julia
-model = Model()
-x = MOI.VariableIndex(1)
-c1 = add_constraint(model, :(\$x^2), MOI.LessThan(1.0))
-c2 = add_constraint(model, :(\$x^2), MOI.LessThan(1.0))
-evaluator = Evaluator(model)
-MOI.initialize(evaluator, Symbol[])
-ordinal_index(evaluator, c2)  # Returns 2
-delete(model, c1)
-evaluator = Evaluator(model)
-MOI.initialize(evaluator, Symbol[])
-ordinal_index(model, c2)  # Returns 1
```

Comment:
This should have been `ordinal_index(evaluator, c2)`. It's amazing just how many mistakes there are if you don't test the examples.